modules/rdcb/README.md

```diff
@@ -41,13 +44,13 @@
 | PatchSchedule | Schedule used to apply patches to the instance | `string` | `"cron(0 6 ? * Sat *)"` | no |
 | PatchSnsTopicArn | SNS Topic used for patch status notifications | `string` | `""` | no |
 | RdcbDnszoneId | Zone to create DNS record for RDCB instance | `string` | `""` | no |
-| RepoBranchPrefixUrl | URL prefix where the repo scripts can be retrieved | `string` | `"https://raw.githubusercontent.com/plus3it/cfn/master"` | no |
+| RemoteAccessScriptsUrl | URL prefix where the repo scripts can be retrieved | `string` | `"https://raw.githubusercontent.com/terraform-aws-remote-access/master"` | no |
```

Comment:
Are my eyes broken, or is this missing the org? Why isn't tfdocs catching this?
```suggestion
| RemoteAccessScriptsUrl | URL prefix where the repo scripts can be retrieved | `string` | `"https://raw.githubusercontent.com/plus3it/terraform-aws-remote-access/master"` | no |
```

Reply:
fixed